watson_developer_cloud/speech_to_text_v1_adapter.py

```diff
@@ -178,7 +186,8 @@ def recognize_using_websocket(self,
             'customization_id': customization_id,
             'acoustic_customization_id': acoustic_customization_id,
             'customization_weight': customization_weight,
-            'base_model_version': base_model_version
+            'base_model_version': base_model_version,
+            'language_customization_id': language_customization_id,
```

Comment:
Does the order of params matter here? do we need to switch the positions of `customization_id` and `language_customization_id`?

Reply:
The order doesnt matter here.